recipes/tableschema-sql/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "tableschema-sql" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 524d5aa4da2fad6a1fd6a455a2c4727f794dcd25b69e765731c2e106d47a664d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Could it be ``noarch`` as well?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  noarch: python
```